pkg/email/templates/temporary_password.html.tmpl

```diff
@@ -0,0 +1,26 @@
+{{- define "title" -}}
+Temporary Password
+{{- end -}}
+
+{{- define "preview" -}}
+A temporary password was requested for your user "{{ .Receiver.Ids.IDString }}".
+{{- end -}}
+
+{{- define "body" -}}
+<p>
+Dear {{ .ReceiverName }},
+</p>
+<p>
+A temporary password was requested for your user "{{ .Receiver.Ids.IDString }}" on {{ .Network.Name }}.
+</p>
+<p>
+You can now visit <a href="{{ .Network.IdentityServerURL }}/update-password?user={{ .Receiver.Ids.IDString }}&current={{ .TemporaryPassword }}">this link</a> to change your password.
```

Comment:
```suggestion
You can now visit <a href="{{ .Network.IdentityServerURL }}/update-password?user={{ .Receiver.Ids.IDString }}&current={{ .TemporaryPassword }}">this link</a> to change your password.<br/><br/>
```

Reply:
Not going to do this. Feel free to rewrite the HTML in a future PR if you feel like it.